site/content/blog/kctrl-release-blog.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "Introducing kctrl, kapp-controller’s native CLI"
+slug: kctrl-release-blog
+date: 2022-03-18
+author: Soumik Majumder
+excerpt: "Learn about the new kctrl and how it makes it easier for users to get up and running with Carvel packages."
+image: /img/logo.svg
+tags: ['Soumik Majumder', 'kapp-controller', 'kctrl', 'carvel']
+---
+
+kapp-controller provides declarative APIs to customise, install and update applications and packages reliably replicating workflows involving commonly used tools. It also allows authors of such workflows to package them and ship different versions of such workflows to consumers. The consumers in turn can consume these worlflows using declarative APIs.
+
+We realised that package consumers who are not comfortable with authoring YAMLs often face a steep learning curve while trying to get them up and running in their environments. How do we make it easier for package consumers to install, configure and updgrade packages? Enter `kctrl`.
+
+## What is `kctrl`?
+`kctrl` is kapp-controller's native CLI which help users effectively interact with and observe custom resources exposed by kapp-controller (App, PackageInstall, PackageRepository, Package CRs). The first release of `kctrl` focuses on helping users getting started with consuming packages with minimum friction. In the next few releases, `kctrl` aims to improve observability of resources created using kapp-controller's declarative APIs, improving the debugging experience and helping users authoring Packages and AppCRs to iterate faster and more efficiently.
```

Comment:
nit: `getting started on consuming packages with minimum` instead of `getting started with consuming packages with minimum`

Reply:
I think the latter makes more sense 🤔 Leaving this in for now